lua/acf/menu/operations/acf_menu.lua

```diff
@@ -219,7 +219,8 @@ do -- Generic Spawner/Linker operation creator
 	--- @param Name string The name of the link type performed by the toolgun (e.g. Weapon, Engine, etc.)
 	--- @param Primary string The type of the entity to be spawned on left click (purely aesthetical)
 	--- @param Secondary string | nil The type of entity to be spawned on shift + right click (purely aesthetical)
-	function ACF.CreateMenuOperation(Name, Primary, Secondary)
+	--- @param OnRightClick table | nil If provided, a table with a Text and Func parameter for when right clicking
```

Comment:
Could we move the conversion operation to R or Shift or something? We will probably want the ability to link baseplates to things later if/when we can get an autotread system going.